src/java.base/share/classes/com/sun/crypto/provider/ChaCha20Cipher.java

```diff
@@ -675,7 +675,7 @@ protected byte[] engineUpdate(byte[] in, int inOfs, int inLen) {
     @Override
     protected int engineUpdate(byte[] in, int inOfs, int inLen,
             byte[] out, int outOfs) throws ShortBufferException {
-        int bytesUpdated = 0;
+        int bytesUpdated;
```

Comment:
Not relevant to this line, but rather down below in the javadoc of engineUpdate(ByteBuffer, ByteBuffer), the javadoc has {@code out} but the variable name is `output`, probably a copy-n-paste error from this method.

Reply:
Fixed.